source/overview/compliance.rst

```diff
@@ -174,11 +174,6 @@ Based on our interpretation of GDPR, it is not required to hide message contents
 2. Mattermost offers :ref:`TLS encryption <email-tls>` to protect communication between the Mattermost server and the SMTP email server.
 3. For those who are uncertain if the first two points cover GDPR compliance, we offer the ability to :ref:`disable notifications completely <email-notification-config>` on your Mattermost server. To use Mattermost in production with no email notifications, you also need to :ref:`disable a "preview mode" notice banner <email-preview-mode-banner-config>`.
 
-What information is shared when I click **Contact us** on a Mattermost Admin Advisor notification?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Clicking **Contact us** in the Mattermost Admin Advisor will send some information to us. This may include the email address and name associated with your Mattermost account as well as the number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number. This information is used to contact you as requested and to help us better understand your needs.
```

Comment:
@harshilsharma63 - After discussing with @jasonblais, it's preferable to note that the Admin Advisor feature has been disabled for transparency, rather than to remove references to the feature from the product documentation. 

Can you reverse this section deletion, please, and then add the following RST-formatted note at the end of the section?

.. note::
    Mattermost `Admin Advisor notices are disabled <https://docs.mattermost.com/administration/notices.html#admin-advisor-notices>`__ in Mattermost Server from v5.35.

Reply:
Done.